fixtures/ios_app_with_frameworks/App/Project.swift

```diff
@@ -1,150 +0,0 @@
-import ProjectDescription
```

Comment:
This file doesn't actually appear to be used, has a syntax error, and burned a lot of time until I figured out it wasn't even being used :(.